ffdc.sh

```diff
@@ -24,7 +28,7 @@ function close_ffdc {
 }
 
 function fail_unless_bastion {
-	oc_project_output=$(oc project ${project_name})
+	oc_project_output=$(oc project ${PROJECT_NAME})
 	if [[ "$?" != "0" ]]	
 	then
 		error "Must be run on the Openshift bastion node with the Openshift command line."
```

Comment:
In the same way, the error message could change to something like "An error occurred when trying to retrieve the namespace.". We are not using a bastion node anymore.